pyControl4/director.py

```diff
@@ -1,11 +1,75 @@
-"""Handles communication with a Control4 Director, and provides functions for getting details about items on the Director.
+"""Handles communication with a Control4 Director, and provides functions for
+   getting details about items on the Director.
 """
 import aiohttp
 import async_timeout
 import json
+import socketio
+import logging
 
 from .error_handling import checkResponseForError
 
+_LOGGER = logging.getLogger(__name__)
+
+
+class C4DirectorNamespace(socketio.AsyncClientNamespace):
```

Comment:
I wonder if we should move this class into a separate .py file and import it into `director.py`. The code in this namespace class seems to be pretty unrelated to the rest of the director functions.